troubleshooting/debugging.md

```diff
@@ -589,6 +590,15 @@ The [tail input](https://docs.fluentbit.io/manual/pipeline/inputs/tail) has a se
 
 We have seen that if there are a large number of files on disk matched by the Fluent Bit tail `Path`, then Fluent Bit may become slow and may even lose logs. This happens because it must processes watch events for each file. Consider setting up log file deletion and set `Ignore_Older` so that Fluent Bit can stop watching older files.
 
+We have a [Log File Deletion example](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/ecs-log-deletion). Setting up log file deletion is required to ensure Fluent Bit tail input can achieve ideal performance. While `Ignore_Older` will ensure that it does not process events for older files, the input will still continuously scan your configured `Path` for all log files that it matches. If you have lots of old files that match your `Path`, this will slow it down.
+
+#### Tail input log loss and log file deletion
```

Comment:
I am not sure I make the connection between log loss and needing to delete log files. From the content in this section. It feels like additional information for `Ignore_Older`

Reply:
Good point. I wanted to put this in the title just because of that one extreme case that showed that lack of log deletion can cause log loss...

Reply:
I edited it to just say perf issues but still keep it linked in the log loss section summary